inference-engine/src/inference_engine/src/compilation_context.cpp

```diff
@@ -97,7 +97,7 @@ std::string NetworkCompilationContext::computeHash(const CNNNetwork& network,
 
     // 1. Serialize
     CNNNetwork net(network);
-    ngraph::pass::Serialize serializer(xml, bin);
+    ov::pass::Serialize serializer(xml, bin);
```

Comment:
not directly related to this PR: @nosovmik how do you think can we switch to IR v11 serializer with all runtime information as a part of the graph, and remove additional cycles where we compute hash on runtime info later in this file?